### PR TITLE
renaming environment variable prefix

### PR DESCRIPTION
**Pull request by ishitam8**

### CODE REVIEW

File: azure-devops/azext_devops/dev/common/const.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 
 # Environment Variables
-CLI_ENV_VARIABLE_PREFIX = 'AZURE_DEVOPS_CLI_'
+CLI_ENV_VARIABLE_PREFIX = 'AZURE_DEVOPS_EXT_'
```
[the remark this answers is not in the document]
Comment (ishitam8) [reply azure-devops/azext_devops/dev/common/const.py]: As suggested by @geverghe , we are avoiding CLI in the names and just going with 'extension'.

[the remark this answers is not in the document]
Comment (geverghe) [reply azure-devops/azext_devops/dev/common/const.py]: So the recommendation is for us to stay consistent - Azure DevOps Extension and Azure CLI. We do not include CLI reference with Azure DevOps

[the remark this answers is not in the document]
Comment (gauravsaralMs) [reply azure-devops/azext_devops/dev/common/const.py]: @geverghe : This still holds good, right?

[the remark this answers is not in the document]
Comment (geverghe) [reply azure-devops/azext_devops/dev/common/const.py]: Yes.

**Pull request merged**